.github/workflows/lint_and_test_go-client.yml

```diff
@@ -36,27 +36,12 @@ on:
 
 # workflow tasks
 jobs:
-  lint:
-    name: Lint
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v2
-        with:
-          fetch-depth: 1
-      - name: Set up Go
-        uses: actions/setup-go@v2
-        with:
-          go-version: 1.14
-      - name: golangci-lint
-        uses: golangci/golangci-lint-action@v3
-        with:
-          version: v1.29
-          working-directory: ./go-client
-
   build:
-    name: Test
-    runs-on: ubuntu-latest
+    name: Test and Lint
+    runs-on: ubuntu:20.04
     steps:
+    - name: Install thrift
+      run: sudo apt-get install -y thrift-compiler
```

Comment:
We do not need to specify the version of the thrift compiler?

Reply:
Yes, so I specify the runner as ubuntu-20.04, on which the thrift-compiler is 0.13.0

Reply:
`sudo apt-get install -y thrift-compiler='0.13.0-*'`

And we use thrift 0.11.0 in java client?

Reply:
tracked by https://github.com/apache/incubator-pegasus/issues/1283